src/main/scala/mesosphere/marathon/core/deployment/impl/DeploymentManagerActor.scala

```diff
@@ -407,17 +406,17 @@ object DeploymentManagerActor {
 
   @SuppressWarnings(Array("MaxParameters"))
   def props(
-    taskTracker: InstanceTracker,
     killService: KillService,
-    launchQueue: LaunchQueue,
-    scheduler: SchedulerActions,
+    launchQueue: LaunchQueue, // TODO(karsten): Remove launch queue and go through scheduling.Scheduler.
```

Comment:
The todo is part of [phase 3](https://jira.mesosphere.com/browse/MARATHON-8292).